pkg/web_app/lib/src/account.dart

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:html';
+import 'dart:js';
+
+import 'package:http/http.dart';
+
+import '_authenticated_client.dart';
+import 'google_auth_js.dart';
+
+bool _initialized = false;
+Element _logoutElem;
+Element _accountInfoElem;
+
+GoogleUser _currentUser;
+
+/// Returns whether the user is currently signed-in.
+bool get isSignedIn => _initialized && _currentUser != null;
+
+/// Returns the currently signed-in user or null.
+GoogleUser get currentUser => _currentUser;
+
+Client _client;
+
+/// The HTTP Client to use with account credentials.
+Client get client {
+  return _client ??=
+      getAuthenticatedClient(currentUser.getAuthResponse(true)?.access_token);
+}
+
+Future<void> setupAccount() async {
+  // Initialization hook that will run after the auth library is loaded and
+  // initialized. Method name is passed in as request parameter when loading
+  // the auth library.
+  context['pubAuthInit'] = () {
+    _initialized = true;
```

Comment:
According to docs: https://developers.google.com/identity/sign-in/web/reference
You need to call `gapi.load('auth2', () {....});`

At-least that's how I read it, see the "Auth setup" section.

Reply:
True... I guess we shall call that, although it is working without it :)

Reply:
Maybe it works because it's cached or something? Or maybe it just always works. I would just prefer to rely on documented behavior :)